Lab 1.ipynb

```diff
@@ -0,0 +1,581 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "k = 2\n",
+    "p = .1\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "x = np.arange(0, 50, 0.5)\n",
+    "y = stats.binom.pmf(k, x, p, loc = 0)\n",
+    "\n",
+    "plt.tick_params(labelsize = 12)\n",
+    "ax.set_ylabel('Probability', fontsize = 12)\n",
+    "ax.set_xlabel('Value', fontsize = 12)\n",
+    "\n",
+    "ax.plot(x,y, 'bo')\n",
+    "\n",
+    "# Much more antisymmetric than the first one, this is due to being more \"zoomed in\" on the central area. If the domain was increased, then it would start to have more of a peak like the first one.\n",
```

Comment:
It would be nice to have your k and p values included in these printouts - as titles of your plots maybe? Would help tell them apart. You could also do them as plt.subplots(1, 3) if you wanted them printed all together for better comparison.